.vscode/settings.json

```diff
@@ -25,19 +25,23 @@
 	// Custom dictionary for 'streetsidesoftware.code-spell-checker' extension.
 	"cSpell.words": [
 		"endregion",
+		"handlecache",
```

Comment:
a lot of the added words here are not words. I'm not sure adding them is a good idea.